ci-operator/config/openshift/logging-view-plugin/openshift-logging-view-plugin-main.yaml

```diff
@@ -0,0 +1,29 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_literal: |
+    FROM cypress/included:3.8.0
+  to: plugin-test
```

Comment:
This should be something like we do for any other logging related image.  see origin-aggregated-logging or cluster-logging-operator or logging-fluentd